### PR TITLE
adding subcommand about show info

### LINKED ISSUE #74: `info` subcommand
Add subcommand to show general info
- Where config file is located at
- Where trove file is located at
-....

### PR DESCRIPTION
**Pull request by kyoto7250**
close #74 

This PR adds `info` commands.

I think this command will be useful in the future when users can customize it.

Thank you in advance.

### CODE REVIEW

File: src/hoard.rs [hunk 1]
```diff
@@ -65,6 +65,24 @@ impl Hoard {
         };
     }
 
+    pub fn show_info(&self) {
+        if let Some(config) = &self.config {
+            if let Some(config_home_path) = &config.config_home_path {
+                println!(
+                    "✨ Where config file is located at {}",
```
[the remark this answers is not in the document]
Comment (Hyde46) [reply src/hoard.rs:72]: Sounds more natural when reading the whole line

File: src/hoard.rs [hunk 2]
```diff
@@ -65,6 +65,24 @@ impl Hoard {
...
+                    config_home_path.display()
+                );
+            }
+
+            if let Some(trove_home_path) = &config.trove_home_path {
+                println!(
+                    "✨ Where trove file is located at {}",
```
Comment (Hyde46) [inline src/hoard.rs:79]: `✨ Tove file is located at {}`
Same here

**Comment by kyoto7250** [comment]
https://github.com/Hyde46/hoard/pull/106#pullrequestreview-947287838

@Hyde46  Thank you for your review 🤩

I fixed [this commit](https://github.com/Hyde46/hoard/pull/106/commits/a20ad60d68a237dabe03ea789a2078b58a61372a).

**Review by Hyde46 — APPROVED** [review]
Looks great, thanks a lot! 😄 🚀 
I will update the Readme and then include it with a new build

**Pull request merged**